core/src/main/java/org/apache/accumulo/core/spi/metrics/FilteringLoggingMRFactory.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.spi.metrics;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.config.MeterFilter;
+
+public class FilteringLoggingMRFactory extends LoggingMeterRegistryFactory {
+
+  @Override
+  public MeterRegistry create(final InitParameters params) {
+    var meterRegistry = super.create(params);
+    MeterFilter myFilter = MeterFilter.denyNameStartsWith("accumulo.gc");
```

Comment:
This looks like the code in a comment I made on #4599.  That comment was intended as an example showing how a user could easily use subclassing and existing micrometer filters to filter metrics. The actual filters a user may want to could vary greatly so if this were added to Accumulo somewhere it would be better to do so as documentation.